2017-11-02 16:13:09.448 UTC [ConnProducer] NewConnection -> ERRO 480 Failed connecting to orderer.example.com:7050 , error: context deadline exceeded
2017-11-02 16:13:09.448 UTC [deliveryClient] connect -> DEBU 481 Connected to 
2017-11-02 16:13:09.448 UTC [deliveryClient] connect -> ERRO 482 Failed obtaining connection: Could not connect to any of the endpoints: [orderer.example.com:7050]
2017-11-02 16:13:09.448 UTC [deliveryClient] try -> WARN 483 Got error: Could not connect to any of the endpoints: [orderer.example.com:7050] at 10 attempt. Ceasing to retry
2017-11-02 16:13:09.448 UTC [blocksProvider] DeliverBlocks -> WARN 484 [testchannelid] Receive error: Attempts (10) or elapsed time (9m1.026757789s) exhausted
